src/main/scala/is/hail/rvd/OrderedRVDPartitioner.scala

```diff
@@ -8,66 +8,144 @@ import org.apache.spark.Partitioner
 class OrderedRVDPartitioner(
   val numPartitions: Int,
   val partitionKey: Array[String], val kType: TStruct,
-  // rangeBounds is partition max, sorted ascending
-  // rangeBounds: Array[pkType]
```

Comment:
Please keep in a comment about the Hail type of rangeBounds.  I found this very useful documentation when developing the RVD stuff.